locale/fi/LC_MESSAGES/django.po

```diff
@@ -382,52 +382,56 @@ msgstr "Seuraavat käyttäjät olivat jo ilmoittautuneet: {users}"
 msgid "Begin by enrolling in a course."
 msgstr "Aloita ilmoittautumalla jollekin kurssille."
 
-#: course/templates/course/_course_menu.html:14
-#: course/templates/course/_course_menu.html:38
+#: course/templates/course/_course_menu.html:12
+msgid "Skip course menu"
+msgstr "Ohita kurssivalikko"
```

Comment:
Usually "kurssivalikko" refers to a listing of courses (or a menu consisting of courses) if I've understood correctly and have the correct impression. This seems to refer to the menu of the course. I'm not sure if the term can be used in that context. However, I'm not sure what would be a better term.

Reply:
I think, this could be called "Skip course navigation" in English, but that translates to "Ohita kurssinavigointi", which doesn't sound very good though.